docs/backplane/guideline.md

```diff
@@ -0,0 +1,103 @@
+
+## Personas
+
+* Managed Cloud Service (MCS)
+   * formerly Customer Experience and Engagement (CEE)
+* Site Reliability Engineer (SRE)
+   * https://landing.google.com/sre/ 
+* Platform SRE (SREP)
+   * SRE in SD primarily responsible for Managed OpenShift Platform
+   * https://source.redhat.com/groups/public/sre 
+* Layered Product SRE (LPSRE)
+   * SRE in SD primarily responsible for Managed Services / Layered Products
+   * formerly MT-SRE and CS-SRE
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/managed_tenants_sre_introduction
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/cssre_introduction 
+* Application SRE (AppSRE)
+   * SRE in SD primarily responsible for Red Hat internal applications and capabilities provided to other Red Hat personas and Red Hat customers
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/app_sre_introduction_new 
+* Hybrid SRE
+   * Engineering teams are SRE for their products in conjunction with other SD SRE teams.
+   * https://source.redhat.com/groups/public/sdsea/hybridsre 
+   * Teams that are the initial focus include HyperShift, ACS, and ODS
+
+## Privilege Escalation
+
+Some personas may elevate permissions beyond the day-to-day permisisons.  All such elevations are audited and require justification and approval from managers after use and must never be used lightly.
+
+Personas permitted to use privilege escalation on customer clusters are:
+* SREP
+* LPSRE
+
+Escalation of privileges uses a specific user to eleavte permissios to `cluster-admin`.  More information on [ops-sop/v4/howto/backplane-elevate-privileges.md](https://github.com/openshift/ops-sop/blob/master/v4/howto/backplane-elevate-privileges.md).
+
+## Day-to-Day
+
+Permissions for day-to-day operations should not require any special permission to utilize.  These permissions follow these guidelines:
+
+* No secret access except where the secret is known to contain no actual secret data.
+* No customer namespace access as the baseline.  Customer namespace access must be justified based on specific service support needs.
+* No write access by default, all write permissions require justification.
+* Read access to cluster scope resources may be granted if those resources do not contain any sensitive customer data.
+* Read access to namespace scoped resources is never granted at a cluster scope.
+   * Exception may be granted for Red Hat managed systems only, where Red Hat is the customer, and such access is justified for day-to-day operations.
+* Existing ClusterRoles are preferred when granting permissions in a namespace.
+   * `view` - general read-only access to core Kubernetes and OpenShift resources, except for Secrets
+   * `admin` - full administrative access to the namespace.
+   * `dedicated-readers` - created by OSD/ROSA and dynamically aggregates view permissions for resources installed by OLM.
```

Comment:
Why is `admin` an option here? Who would leverage this and how?

Reply:
We do for LPSRE (see [example](https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/lpsre/20-lpsre-addon-operator.SubjectPermission.yml#L11-L12)). This is for legacy purposes and should eventually be removed once support requirements are well-defined.